content/v1.0-docs/configuration/external.md

```diff
@@ -35,6 +35,8 @@ authors are as follows:
   to request certificates signed by
   [Cloudflare Origin CA](https://developers.cloudflare.com/ssl/origin-configuration/origin-ca)
   to enable TLS between Cloudflare edge and your Kubernetes workloads.
+- [ncm-issuer](https://github.com/nokia/ncm-issuer): Requests certificates from the [Nokia](https://www.nokia.com/) [Netguard Certificate Manager](https://www.nokia.com/networks/security-portfolio/netguard/certificate-manager)
```

Comment:
Remove, since this page is for a previous release of the website.

Same goes for the other pages under the 'content/vx.x-docs' directories, we only want to change the current page.

Reply:
okay, thanks! :D